pkg/autodiscovery/listeners/kube_endpoints_test.go

```diff
@@ -99,9 +95,7 @@ func TestProcessEndpoints(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, []string{"kube_service:myservice", "kube_namespace:default", "kube_endpoint_ip:10.0.0.2", "foo:bar"}, tags)
 
-	eps = processEndpoints(kep, false, []string{"foo:bar"})
-	assert.Equal(t, integration.After, eps[0].GetCreationTime())
-	assert.Equal(t, integration.After, eps[1].GetCreationTime())
+	eps = processEndpoints(kep, []string{"foo:bar"})
```

Comment:
This looks a bit odd, although understandable in the context of this patch. Would it make sense to add a comment here, why we don't assert anything about eps? (or just remove this line altogether).

Reply:
I'll take a look -- thanks.  A lot of the test changes were made with vim macros, and I may have missed one or two that still pass but where things no longer make sense.